api/src/main/resources/api.yaml

```diff
@@ -435,6 +436,141 @@ paths:
           description: if house was removed
         '400':
           description: if params are invalid
+  /houses:
+    get:
+      tags:
+        - Houses
+      description: List all houses of the community given a community id
+      operationId: listAllHouses
+      parameters:
+        - in: query
+          name: pageable
+          required: false
+          schema:
+            $ref: '#/components/schemas/Pageable'
+      responses:
+        '200':
+          description: if community exists
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetHouseDetailsResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/GetHouseDetailsResponse'
+        '404':
```

Comment:
`/houses` will never return 404, please delete that case from yaml

Reply:
Fixed!